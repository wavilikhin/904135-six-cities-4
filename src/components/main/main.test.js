import React from 'react';
import renderer from 'react-test-renderer';
import Main from './main.jsx';

const offersDataArray = [
  {
    quality: `Premium`,
    image: `img/apartment-01.jpg`,
    priceValue: `\u20AC120`,
    priceText: ` \u2215\u0020night`,
    name: `Beautiful & luxurious apartment at great location`,
    type: `Apartment`,
    coords: [52.3909553943508, 4.85309666406198],
  },
  {
    quality: `Premium`,
    image: `img/room.jpg`,
    priceValue: `\u20AC110`,
    priceText: ` \u2215\u0020night`,
    name: `Just a nice place`,
    type: `Room`,
    coords: [52.369553943508, 4.85309666406198],
  },
  {
    quality: `Premium`,
    image: `img/apartment-02.jpg`,
    priceValue: `\u20AC100`,
    priceText: ` \u2215\u0020night`,
    name: `Wood and stone place`,
    type: `Apartment`,
    coords: [52.3909553943508, 4.929309666406198],
  },
  {
    quality: `Premium`,
    image: `img/apartment-03.jpg`,
    priceValue: `\u20AC150`,
    priceText: ` \u2215\u0020night`,
    name: `Canal View Prinsengracht`,
    type: `Apartment`,
    coords: [52.3809553943508, 4.939309666406198],
  },
  {
    quality: `Premium`,
    image: `img/room.jpg`,
    priceValue: `\u20AC80`,
    priceText: ` \u2215\u0020night`,
    name: `Nice, cozy, warm big bed apartment`,
    type: `Room`,
    coords: [52.3709553943508, 4.89309666406198],
  },
];

describe(`Main component snapshot test`, () => {
  it(`Main component should render main page
      with number of awaliable places = 5,
      and places infos`, () => {
    const tree = renderer
      .create(<Main offersDataArray={offersDataArray} />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
