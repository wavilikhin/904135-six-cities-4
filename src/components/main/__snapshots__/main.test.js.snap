// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main component snapshot test Main component should render "main" page
      with 11 avaliable places
      and places infos 1`] = `
<div
  className="page page--gray page--main"
>
  <header
    className="header"
  >
    <div
      className="container"
    >
      <div
        className="header__wrapper"
      >
        <div
          className="header__left"
        >
          <a
            className="header__logo-link header__logo-link--active"
          >
            <img
              alt="6 cities logo"
              className="header__logo"
              height="41"
              src="img/logo.svg"
              width="81"
            />
          </a>
        </div>
        <nav
          className="header__nav"
        >
          <ul
            className="header__nav-list"
          >
            <li
              className="header__nav-item user"
            >
              <a
                href="/favorites"
                onClick={[Function]}
              >
                <span>
                  
                </span>
              </a>
            </li>
          </ul>
        </nav>
      </div>
    </div>
  </header>
  <main
    className="page__main page__main--index"
  >
    <h1
      className="visually-hidden"
    >
      Cities
    </h1>
    <div
      className="tabs"
    >
      <section
        className="locations container"
      >
        <ul
          className="locations__list tabs__list"
        >
          <li
            className="locations__item"
          >
            <a
              className="locations__item-link tabs__item tabs__item--active"
              href="#"
              onClick={[Function]}
            >
              <span>
                Amsterdam
              </span>
            </a>
          </li>
          <li
            className="locations__item"
          >
            <a
              className="locations__item-link tabs__item "
              href="#"
              onClick={[Function]}
            >
              <span>
                Hamburg
              </span>
            </a>
          </li>
          <li
            className="locations__item"
          >
            <a
              className="locations__item-link tabs__item "
              href="#"
              onClick={[Function]}
            >
              <span>
                Paris
              </span>
            </a>
          </li>
          <li
            className="locations__item"
          >
            <a
              className="locations__item-link tabs__item "
              href="#"
              onClick={[Function]}
            >
              <span>
                Dusseldorf
              </span>
            </a>
          </li>
          <li
            className="locations__item"
          >
            <a
              className="locations__item-link tabs__item "
              href="#"
              onClick={[Function]}
            >
              <span>
                Brussels
              </span>
            </a>
          </li>
          <li
            className="locations__item"
          >
            <a
              className="locations__item-link tabs__item "
              href="#"
              onClick={[Function]}
            >
              <span>
                Cologne
              </span>
            </a>
          </li>
        </ul>
      </section>
    </div>
    <div
      className="cities"
    >
      <div
        className="cities__places-container container"
      >
        <section
          className="cities__places places"
        >
          <h2
            className="visually-hidden"
          >
            Places
          </h2>
          <b
            className="places__found"
          >
            11
             places to stay in 
            Amsterdam
          </b>
          <form
            action="#"
            className="places__sorting"
            method="get"
          >
            <span
              className="places__sorting-caption"
            >
              Sort by
            </span>
            <span
              className="places__sorting-type"
              tabIndex="0"
            >
              Popular
              <svg
                className="places__sorting-arrow"
                height="4"
                width="7"
              >
                <use
                  xlinkHref="#icon-arrow-select"
                />
              </svg>
            </span>
            <ul
              className="places__options places__options--custom places__options--opened"
            >
              <li
                className="places__option places__option--active"
                tabIndex="0"
              >
                Popular
              </li>
              <li
                className="places__option"
                tabIndex="0"
              >
                Price: low to high
              </li>
              <li
                className="places__option"
                tabIndex="0"
              >
                Price: high to low
              </li>
              <li
                className="places__option"
                tabIndex="0"
              >
                Top rated first
              </li>
            </ul>
          </form>
          <div
            className="cities__places-list places__list tabs__content"
          >
            <article
              className="cities__place-card place-card"
            >
              <div
                className="cities__image-wrapper place-card__image-wrapper"
              >
                <a
                  href="#"
                >
                  <img
                    alt="Place image"
                    className="place-card__image"
                    height="200"
                    src="https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/15.jpg"
                    width="260"
                  />
                </a>
              </div>
              <div
                className="place-card__info"
              >
                <div
                  className="place-card__price-wrapper"
                >
                  <div
                    className="place-card__price"
                  >
                    <b
                      className="place-card__price-value"
                    >
                      €
                      813
                    </b>
                    <span
                      className="place-card__price-text"
                    >
                       
                      ∕ night
                    </span>
                  </div>
                  <button
                    className="place-card__bookmark-button button "
                    onClick={[Function]}
                    type="button"
                  >
                    <svg
                      className="place-card__bookmark-icon"
                      height="19"
                      width="18"
                    >
                      <use
                        xlinkHref="#icon-bookmark"
                      />
                    </svg>
                    <span
                      className="visually-hidden"
                    >
                      To bookmarks
                    </span>
                  </button>
                </div>
                <div
                  className="place-card__rating rating"
                >
                  <div
                    className="place-card__stars rating__stars"
                  >
                    <span
                      style={
                        Object {
                          "width": "48%",
                        }
                      }
                    />
                    <span
                      className="visually-hidden"
                    >
                      Rating
                    </span>
                  </div>
                </div>
                <h2
                  className="place-card__name"
                >
                  <a
                    href="/offer/1"
                    id="linkToOffer"
                    onClick={[Function]}
                  >
                    Wood and stone place
                  </a>
                </h2>
                <p
                  className="place-card__type"
                >
                  house
                </p>
              </div>
            </article>
            <article
              className="cities__place-card place-card"
            >
              <div
                className="cities__image-wrapper place-card__image-wrapper"
              >
                <a
                  href="#"
                >
                  <img
                    alt="Place image"
                    className="place-card__image"
                    height="200"
                    src="https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/10.jpg"
                    width="260"
                  />
                </a>
              </div>
              <div
                className="place-card__info"
              >
                <div
                  className="place-card__price-wrapper"
                >
                  <div
                    className="place-card__price"
                  >
                    <b
                      className="place-card__price-value"
                    >
                      €
                      101
                    </b>
                    <span
                      className="place-card__price-text"
                    >
                       
                      ∕ night
                    </span>
                  </div>
                  <button
                    className="place-card__bookmark-button button "
                    onClick={[Function]}
                    type="button"
                  >
                    <svg
                      className="place-card__bookmark-icon"
                      height="19"
                      width="18"
                    >
                      <use
                        xlinkHref="#icon-bookmark"
                      />
                    </svg>
                    <span
                      className="visually-hidden"
                    >
                      To bookmarks
                    </span>
                  </button>
                </div>
                <div
                  className="place-card__rating rating"
                >
                  <div
                    className="place-card__stars rating__stars"
                  >
                    <span
                      style={
                        Object {
                          "width": "70%",
                        }
                      }
                    />
                    <span
                      className="visually-hidden"
                    >
                      Rating
                    </span>
                  </div>
                </div>
                <h2
                  className="place-card__name"
                >
                  <a
                    href="/offer/5"
                    id="linkToOffer"
                    onClick={[Function]}
                  >
                    Perfectly located Castro
                  </a>
                </h2>
                <p
                  className="place-card__type"
                >
                  room
                </p>
              </div>
            </article>
            <article
              className="cities__place-card place-card"
            >
              <div
                className="cities__image-wrapper place-card__image-wrapper"
              >
                <a
                  href="#"
                >
                  <img
                    alt="Place image"
                    className="place-card__image"
                    height="200"
                    src="https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/17.jpg"
                    width="260"
                  />
                </a>
              </div>
              <div
                className="place-card__info"
              >
                <div
                  className="place-card__price-wrapper"
                >
                  <div
                    className="place-card__price"
                  >
                    <b
                      className="place-card__price-value"
                    >
                      €
                      423
                    </b>
                    <span
                      className="place-card__price-text"
                    >
                       
                      ∕ night
                    </span>
                  </div>
                  <button
                    className="place-card__bookmark-button button "
                    onClick={[Function]}
                    type="button"
                  >
                    <svg
                      className="place-card__bookmark-icon"
                      height="19"
                      width="18"
                    >
                      <use
                        xlinkHref="#icon-bookmark"
                      />
                    </svg>
                    <span
                      className="visually-hidden"
                    >
                      To bookmarks
                    </span>
                  </button>
                </div>
                <div
                  className="place-card__rating rating"
                >
                  <div
                    className="place-card__stars rating__stars"
                  >
                    <span
                      style={
                        Object {
                          "width": "94%",
                        }
                      }
                    />
                    <span
                      className="visually-hidden"
                    >
                      Rating
                    </span>
                  </div>
                </div>
                <h2
                  className="place-card__name"
                >
                  <a
                    href="/offer/14"
                    id="linkToOffer"
                    onClick={[Function]}
                  >
                    Penthouse, 4-5 rooms + 5 balconies
                  </a>
                </h2>
                <p
                  className="place-card__type"
                >
                  hotel
                </p>
              </div>
            </article>
            <article
              className="cities__place-card place-card"
            >
              <div
                className="cities__image-wrapper place-card__image-wrapper"
              >
                <a
                  href="#"
                >
                  <img
                    alt="Place image"
                    className="place-card__image"
                    height="200"
                    src="https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/9.jpg"
                    width="260"
                  />
                </a>
              </div>
              <div
                className="place-card__info"
              >
                <div
                  className="place-card__price-wrapper"
                >
                  <div
                    className="place-card__price"
                  >
                    <b
                      className="place-card__price-value"
                    >
                      €
                      134
                    </b>
                    <span
                      className="place-card__price-text"
                    >
                       
                      ∕ night
                    </span>
                  </div>
                  <button
                    className="place-card__bookmark-button button "
                    onClick={[Function]}
                    type="button"
                  >
                    <svg
                      className="place-card__bookmark-icon"
                      height="19"
                      width="18"
                    >
                      <use
                        xlinkHref="#icon-bookmark"
                      />
                    </svg>
                    <span
                      className="visually-hidden"
                    >
                      To bookmarks
                    </span>
                  </button>
                </div>
                <div
                  className="place-card__rating rating"
                >
                  <div
                    className="place-card__stars rating__stars"
                  >
                    <span
                      style={
                        Object {
                          "width": "58%",
                        }
                      }
                    />
                    <span
                      className="visually-hidden"
                    >
                      Rating
                    </span>
                  </div>
                </div>
                <h2
                  className="place-card__name"
                >
                  <a
                    href="/offer/15"
                    id="linkToOffer"
                    onClick={[Function]}
                  >
                    Loft Studio in the Central Area
                  </a>
                </h2>
                <p
                  className="place-card__type"
                >
                  house
                </p>
              </div>
            </article>
            <article
              className="cities__place-card place-card"
            >
              <div
                className="cities__image-wrapper place-card__image-wrapper"
              >
                <a
                  href="#"
                >
                  <img
                    alt="Place image"
                    className="place-card__image"
                    height="200"
                    src="https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/5.jpg"
                    width="260"
                  />
                </a>
              </div>
              <div
                className="place-card__info"
              >
                <div
                  className="place-card__price-wrapper"
                >
                  <div
                    className="place-card__price"
                  >
                    <b
                      className="place-card__price-value"
                    >
                      €
                      820
                    </b>
                    <span
                      className="place-card__price-text"
                    >
                       
                      ∕ night
                    </span>
                  </div>
                  <button
                    className="place-card__bookmark-button button "
                    onClick={[Function]}
                    type="button"
                  >
                    <svg
                      className="place-card__bookmark-icon"
                      height="19"
                      width="18"
                    >
                      <use
                        xlinkHref="#icon-bookmark"
                      />
                    </svg>
                    <span
                      className="visually-hidden"
                    >
                      To bookmarks
                    </span>
                  </button>
                </div>
                <div
                  className="place-card__rating rating"
                >
                  <div
                    className="place-card__stars rating__stars"
                  >
                    <span
                      style={
                        Object {
                          "width": "72%",
                        }
                      }
                    />
                    <span
                      className="visually-hidden"
                    >
                      Rating
                    </span>
                  </div>
                </div>
                <h2
                  className="place-card__name"
                >
                  <a
                    href="/offer/26"
                    id="linkToOffer"
                    onClick={[Function]}
                  >
                    The Pondhouse - A Magical Place
                  </a>
                </h2>
                <p
                  className="place-card__type"
                >
                  house
                </p>
              </div>
            </article>
            <article
              className="cities__place-card place-card"
            >
              <div
                className="cities__image-wrapper place-card__image-wrapper"
              >
                <a
                  href="#"
                >
                  <img
                    alt="Place image"
                    className="place-card__image"
                    height="200"
                    src="https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/12.jpg"
                    width="260"
                  />
                </a>
              </div>
              <div
                className="place-card__info"
              >
                <div
                  className="place-card__price-wrapper"
                >
                  <div
                    className="place-card__price"
                  >
                    <b
                      className="place-card__price-value"
                    >
                      €
                      786
                    </b>
                    <span
                      className="place-card__price-text"
                    >
                       
                      ∕ night
                    </span>
                  </div>
                  <button
                    className="place-card__bookmark-button button "
                    onClick={[Function]}
                    type="button"
                  >
                    <svg
                      className="place-card__bookmark-icon"
                      height="19"
                      width="18"
                    >
                      <use
                        xlinkHref="#icon-bookmark"
                      />
                    </svg>
                    <span
                      className="visually-hidden"
                    >
                      To bookmarks
                    </span>
                  </button>
                </div>
                <div
                  className="place-card__rating rating"
                >
                  <div
                    className="place-card__stars rating__stars"
                  >
                    <span
                      style={
                        Object {
                          "width": "48%",
                        }
                      }
                    />
                    <span
                      className="visually-hidden"
                    >
                      Rating
                    </span>
                  </div>
                </div>
                <h2
                  className="place-card__name"
                >
                  <a
                    href="/offer/29"
                    id="linkToOffer"
                    onClick={[Function]}
                  >
                    Beautiful & luxurious apartment at great location
                  </a>
                </h2>
                <p
                  className="place-card__type"
                >
                  house
                </p>
              </div>
            </article>
            <article
              className="cities__place-card place-card"
            >
              <div
                className="cities__image-wrapper place-card__image-wrapper"
              >
                <a
                  href="#"
                >
                  <img
                    alt="Place image"
                    className="place-card__image"
                    height="200"
                    src="https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/2.jpg"
                    width="260"
                  />
                </a>
              </div>
              <div
                className="place-card__info"
              >
                <div
                  className="place-card__price-wrapper"
                >
                  <div
                    className="place-card__price"
                  >
                    <b
                      className="place-card__price-value"
                    >
                      €
                      448
                    </b>
                    <span
                      className="place-card__price-text"
                    >
                       
                      ∕ night
                    </span>
                  </div>
                  <button
                    className="place-card__bookmark-button button "
                    onClick={[Function]}
                    type="button"
                  >
                    <svg
                      className="place-card__bookmark-icon"
                      height="19"
                      width="18"
                    >
                      <use
                        xlinkHref="#icon-bookmark"
                      />
                    </svg>
                    <span
                      className="visually-hidden"
                    >
                      To bookmarks
                    </span>
                  </button>
                </div>
                <div
                  className="place-card__rating rating"
                >
                  <div
                    className="place-card__stars rating__stars"
                  >
                    <span
                      style={
                        Object {
                          "width": "70%",
                        }
                      }
                    />
                    <span
                      className="visually-hidden"
                    >
                      Rating
                    </span>
                  </div>
                </div>
                <h2
                  className="place-card__name"
                >
                  <a
                    href="/offer/46"
                    id="linkToOffer"
                    onClick={[Function]}
                  >
                    Nice, cozy, warm big bed apartment
                  </a>
                </h2>
                <p
                  className="place-card__type"
                >
                  apartment
                </p>
              </div>
            </article>
            <article
              className="cities__place-card place-card"
            >
              <div
                className="cities__image-wrapper place-card__image-wrapper"
              >
                <a
                  href="#"
                >
                  <img
                    alt="Place image"
                    className="place-card__image"
                    height="200"
                    src="https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/6.jpg"
                    width="260"
                  />
                </a>
              </div>
              <div
                className="place-card__info"
              >
                <div
                  className="place-card__price-wrapper"
                >
                  <div
                    className="place-card__price"
                  >
                    <b
                      className="place-card__price-value"
                    >
                      €
                      139
                    </b>
                    <span
                      className="place-card__price-text"
                    >
                       
                      ∕ night
                    </span>
                  </div>
                  <button
                    className="place-card__bookmark-button button "
                    onClick={[Function]}
                    type="button"
                  >
                    <svg
                      className="place-card__bookmark-icon"
                      height="19"
                      width="18"
                    >
                      <use
                        xlinkHref="#icon-bookmark"
                      />
                    </svg>
                    <span
                      className="visually-hidden"
                    >
                      To bookmarks
                    </span>
                  </button>
                </div>
                <div
                  className="place-card__rating rating"
                >
                  <div
                    className="place-card__stars rating__stars"
                  >
                    <span
                      style={
                        Object {
                          "width": "42%",
                        }
                      }
                    />
                    <span
                      className="visually-hidden"
                    >
                      Rating
                    </span>
                  </div>
                </div>
                <h2
                  className="place-card__name"
                >
                  <a
                    href="/offer/48"
                    id="linkToOffer"
                    onClick={[Function]}
                  >
                    The Joshua Tree House
                  </a>
                </h2>
                <p
                  className="place-card__type"
                >
                  hotel
                </p>
              </div>
            </article>
            <article
              className="cities__place-card place-card"
            >
              <div
                className="cities__image-wrapper place-card__image-wrapper"
              >
                <a
                  href="#"
                >
                  <img
                    alt="Place image"
                    className="place-card__image"
                    height="200"
                    src="https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/17.jpg"
                    width="260"
                  />
                </a>
              </div>
              <div
                className="place-card__info"
              >
                <div
                  className="place-card__price-wrapper"
                >
                  <div
                    className="place-card__price"
                  >
                    <b
                      className="place-card__price-value"
                    >
                      €
                      183
                    </b>
                    <span
                      className="place-card__price-text"
                    >
                       
                      ∕ night
                    </span>
                  </div>
                  <button
                    className="place-card__bookmark-button button "
                    onClick={[Function]}
                    type="button"
                  >
                    <svg
                      className="place-card__bookmark-icon"
                      height="19"
                      width="18"
                    >
                      <use
                        xlinkHref="#icon-bookmark"
                      />
                    </svg>
                    <span
                      className="visually-hidden"
                    >
                      To bookmarks
                    </span>
                  </button>
                </div>
                <div
                  className="place-card__rating rating"
                >
                  <div
                    className="place-card__stars rating__stars"
                  >
                    <span
                      style={
                        Object {
                          "width": "46%",
                        }
                      }
                    />
                    <span
                      className="visually-hidden"
                    >
                      Rating
                    </span>
                  </div>
                </div>
                <h2
                  className="place-card__name"
                >
                  <a
                    href="/offer/69"
                    id="linkToOffer"
                    onClick={[Function]}
                  >
                    Penthouse, 4-5 rooms + 5 balconies
                  </a>
                </h2>
                <p
                  className="place-card__type"
                >
                  room
                </p>
              </div>
            </article>
            <article
              className="cities__place-card place-card"
            >
              <div
                className="place-card__mark"
              >
                <span>
                  Premium
                </span>
              </div>
              <div
                className="cities__image-wrapper place-card__image-wrapper"
              >
                <a
                  href="#"
                >
                  <img
                    alt="Place image"
                    className="place-card__image"
                    height="200"
                    src="https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/6.jpg"
                    width="260"
                  />
                </a>
              </div>
              <div
                className="place-card__info"
              >
                <div
                  className="place-card__price-wrapper"
                >
                  <div
                    className="place-card__price"
                  >
                    <b
                      className="place-card__price-value"
                    >
                      €
                      215
                    </b>
                    <span
                      className="place-card__price-text"
                    >
                       
                      ∕ night
                    </span>
                  </div>
                  <button
                    className="place-card__bookmark-button button "
                    onClick={[Function]}
                    type="button"
                  >
                    <svg
                      className="place-card__bookmark-icon"
                      height="19"
                      width="18"
                    >
                      <use
                        xlinkHref="#icon-bookmark"
                      />
                    </svg>
                    <span
                      className="visually-hidden"
                    >
                      To bookmarks
                    </span>
                  </button>
                </div>
                <div
                  className="place-card__rating rating"
                >
                  <div
                    className="place-card__stars rating__stars"
                  >
                    <span
                      style={
                        Object {
                          "width": "64%",
                        }
                      }
                    />
                    <span
                      className="visually-hidden"
                    >
                      Rating
                    </span>
                  </div>
                </div>
                <h2
                  className="place-card__name"
                >
                  <a
                    href="/offer/73"
                    id="linkToOffer"
                    onClick={[Function]}
                  >
                    Wood and stone place
                  </a>
                </h2>
                <p
                  className="place-card__type"
                >
                  apartment
                </p>
              </div>
            </article>
            <article
              className="cities__place-card place-card"
            >
              <div
                className="cities__image-wrapper place-card__image-wrapper"
              >
                <a
                  href="#"
                >
                  <img
                    alt="Place image"
                    className="place-card__image"
                    height="200"
                    src="https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/9.jpg"
                    width="260"
                  />
                </a>
              </div>
              <div
                className="place-card__info"
              >
                <div
                  className="place-card__price-wrapper"
                >
                  <div
                    className="place-card__price"
                  >
                    <b
                      className="place-card__price-value"
                    >
                      €
                      793
                    </b>
                    <span
                      className="place-card__price-text"
                    >
                       
                      ∕ night
                    </span>
                  </div>
                  <button
                    className="place-card__bookmark-button button "
                    onClick={[Function]}
                    type="button"
                  >
                    <svg
                      className="place-card__bookmark-icon"
                      height="19"
                      width="18"
                    >
                      <use
                        xlinkHref="#icon-bookmark"
                      />
                    </svg>
                    <span
                      className="visually-hidden"
                    >
                      To bookmarks
                    </span>
                  </button>
                </div>
                <div
                  className="place-card__rating rating"
                >
                  <div
                    className="place-card__stars rating__stars"
                  >
                    <span
                      style={
                        Object {
                          "width": "84%",
                        }
                      }
                    />
                    <span
                      className="visually-hidden"
                    >
                      Rating
                    </span>
                  </div>
                </div>
                <h2
                  className="place-card__name"
                >
                  <a
                    href="/offer/85"
                    id="linkToOffer"
                    onClick={[Function]}
                  >
                    Tile House
                  </a>
                </h2>
                <p
                  className="place-card__type"
                >
                  house
                </p>
              </div>
            </article>
            ;
          </div>
        </section>
        <div
          className="cities__right-section"
        >
          <section
            className="cities__map map"
          >
            <div
              id="map"
              style={
                Object {
                  "height": "100%",
                }
              }
            />
          </section>
        </div>
      </div>
    </div>
  </main>
</div>
`;
